### PR TITLE
Add video integration guides for 100ms

### PR DESCRIPTION
**Pull request by skydoves**
### 🎯 Goal

Add video integration guides for [100ms SDK](https://www.100ms.live/).

#### General
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Assigned a person / code owner group (required)
- [x] PR targets the `develop` branch

#### Code & documentation
- [x] Comparison screenshots added for visual changes
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### 🎉 GIF

gif

### CODE REVIEW

File: docusaurus/docs/Android/02-client/06-guides/10-video-100ms.mdx [hunk 1]
```diff
@@ -0,0 +1,941 @@
...
+There are only a few setup steps to go through in the Stream dashboard and this guide details all of them:
+
+1. Head over to the [Dashboard](https://dashboard.getstream.io) and login
+2. Create a new app or select your app by name
+3. In the sidebar on the left, select **Ext. Video Integration**
+4. Make sure the **100ms** tab is selected
+
+This is the screen that you navigated to:
+
+![View of the Stream Dashboard when an app was selected with the external video integration.](../../../assets/hms-guide-before.png)
```
Comment (leandroBorgesFerreira) [inline docusaurus/docs/Android/02-client/06-guides/10-video-100ms.mdx:56]: I believe the correct path is: `../../assets/hms-guide-before.png`

Comment (leandroBorgesFerreira) [reply docusaurus/docs/Android/02-client/06-guides/10-video-100ms.mdx:56]: The same goes for all the images. I tested with: 
```
cd [path/to/folder/of/mdx-document]
open [path/of/link/of/image]
```

The command `open` only works when removing one `../`

Comment (skydoves) [reply docusaurus/docs/Android/02-client/06-guides/10-video-100ms.mdx]: Nice catch! Thanks for check things ❤️  
Done: [f589c65](https://github.com/GetStream/stream-chat-android/pull/4181/commits/f589c65353b7c5296f4a30f331fed513d67e8866)

File: docusaurus/docs/Android/02-client/06-guides/10-video-100ms.mdx [hunk 2]
```diff
@@ -0,0 +1,941 @@
...
+```
+
+Next, add the dependencies below to your app-level **build.gradle** file:
+
+```groovy {2, 5}
+// 100ms SDK
+implementation "com.github.100mslive.android-sdk:lib:2.3.4"
+
+// Stream chat SDK
+implementation "io.getstream:stream-chat-android-ui-components:5.10.0"
```
Comment (leandroBorgesFerreira) [inline docusaurus/docs/Android/02-client/06-guides/10-video-100ms.mdx:95]:
```suggestion
implementation "io.getstream:stream-chat-android-ui-components:5.11.0"
```

Comment (skydoves) [reply docusaurus/docs/Android/02-client/06-guides/10-video-100ms.mdx]: done: [01b1344](https://github.com/GetStream/stream-chat-android/pull/4181/commits/01b134496163fd438b10a6c51a9260f45d58abcd)

**Comment by adasiewiczr** [comment]
Let's merge this PR directly to `main`

**Comment by skydoves** [comment]
@adasiewiczr Could you check if it's fine to merge this PR against the main branch? It seems to include other commits.

**Comment by adasiewiczr** [comment]
@skydoves let's merge only the documentation and exclude other commits

**Comment by skydoves** [comment]
@adasiewiczr I created a new PR by cherry-picking needed commits: https://github.com/GetStream/stream-chat-android/pull/4237

**Pull request closed without merging**